src/apps/tco2_dashboard/retirement_trends/impl/retirement_trends_pool.py

```diff
@@ -411,6 +414,54 @@ def modify_klima_token_retirements_df(self, df):
         )
         df['View on PolygonScan'] = '[Click Here](' + df['View on PolygonScan'] + ')'
 
+        df['Project_num'] = df['Project'].apply(
+            lambda x: pd.Series(str(x).split("-"))
+            )[1]
+
+        df.Project_num.fillna('N/A', inplace=True)
+
+        verra_l = 'https://registry.verra.org/app/projectDetail/VCS/'
+
+        df['Project_Link'] = verra_l
+
+        df["Project_Link"] = df["Project_Link"] + df["Project_num"]
+
+        missing_condition_1 = df['Project_Link'].str.match(
+            'https://registry.verra.org/app/projectDetail/VCS/N/A')
+
+        df['Project_Link'] = np.where(
+            missing_condition_1, "N/A", df['Project_Link']
+            )
+
+        missing_condition_2 = df['Project_num'].str.match("N/A")
```

Comment:
This seems redundant - the only way you'd en up with a link formatting like the one above (with `/N/A`) would be if missing_condition_2 is true... so why do we need both?

Reply:
Yh we don't need both, I've amended the code to have only one.